packages/angular_devkit/build_angular/src/protractor/index.ts

```diff
@@ -17,12 +17,23 @@ import * as url from 'url';
 import { runModuleAsObservableFork } from '../utils';
 import { Schema as ProtractorBuilderOptions } from './schema';
 
+interface JasmineNodeOpts {
+  jasmineNodeOpts: {
+    grep?: string;
+    invertGrep?: boolean;
+  };
+}
+
 function runProtractor(root: string, options: ProtractorBuilderOptions): Promise<BuilderOutput> {
-  const additionalProtractorConfig: Partial<ProtractorBuilderOptions> = {
+  const additionalProtractorConfig: Partial<ProtractorBuilderOptions> & Partial<JasmineNodeOpts> = {
```

Comment:
I suppose the mistake here is that this was never `Partial<ProtractorBuilderOptions>` to begin with, and was actually a different type, one for protractor itself.

This isn't super pertinent to the PR content itself, it's just a comment.